types/sanitize-html/index.d.ts

```diff
@@ -6,9 +6,10 @@
 //                 Rinze de Laat <https://github.com/biermeester>
 //                 Will Gibson <https://github.com/WillGibson>
 //                 A penguin <https://github.com/sirMerr>
+//                 Johan Davidsson <https://github.com/johandavidson>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
-import {Options} from "htmlparser2";
+import {ParserOptions} from "htmlparser2";
```

Comment:
As @LinusU  commented at https://github.com/DefinitelyTyped/DefinitelyTyped/pull/33921/files#r266197648, this could be a breaking change. I'm fine with it from a personal perspective, but I'm not certain what the implications would be for everyone using `sanitize-html`, especially if their test coverage is not up to scratch.

I can't do it right now, but will try and run these changes through our tests in the codebase where we use  `sanitize-html`.

Reply:
Ok, lets go the safer way and use the deprecated Options. Then we won't need to change those other modules. I'll change.

Reply:
About TypeScript 2.1: If we add the requirement then also all modules referencing this module needs to add the same requirement. I think that would be too much of a change right now.